core/src/main/resources/error/error-classes.json

```diff
@@ -657,6 +657,59 @@
       "Detected an incompatible DataSourceRegister. Please remove the incompatible library from classpath or upgrade it. Error: <message>"
     ]
   },
+  "INCOMPATIBLE_DATA_FOR_TABLE" : {
+    "message" : [
+      "Cannot write incompatible data for table <tableName>:"
+    ],
+    "subClass" : {
+      "AMBIGUOUS_COLUMN_NAME" : {
+        "message" : [
+          "Ambiguous column name in the input data: <colPath>."
+        ]
+      },
+      "CANNOT_FIND_DATA" : {
+        "message" : [
+          "Cannot find data for output column <colPath>."
+        ]
+      },
+      "CANNOT_SAFELY_CAST" : {
+        "message" : [
+          "Cannot safely cast <colPath>: <from> to <to>."
+        ]
+      },
+      "EXTRA_STRUCT_FIELDS" : {
+        "message" : [
+          "Cannot write extra fields to struct <colPath>: <extraCols>."
+        ]
+      },
+      "NULLABLE_ARRAY_ELEMENTS" : {
+        "message" : [
+          "Cannot write nullable elements to array of non-nulls: <colPath>."
+        ]
+      },
+      "NULLABLE_COLUMN" : {
+        "message" : [
+          "Cannot write nullable values to non-null column <colPath>."
+        ]
+      },
+      "NULLABLE_MAP_VALUES" : {
+        "message" : [
+          "Cannot write nullable elements to array of non-nulls: <colPath>."
```

Comment:
`to array`? Should map values, no?

Reply:
Yeah, my mistake. Thanks for the correction!